python/cucim/src/cucim/core/operations/spatial/tests/test_rotate.py

```diff
@@ -0,0 +1,24 @@
+import os
+
+import cupy
+import numpy as np
+from PIL import Image
+
+import cucim.core.operations.spatial as spt
+
+img = Image.open(os.path.join(os.path.abspath(os.path.dirname(__file__)), "1.png"))
```

Comment:
What is the source of this dice image? (i.e. is it public domain?)

Reply:
If there is any rights issue it should be easy enough to just use one of the images bundled with scikit-image.

e.g. for an RGB image:
`skimage.data.astronaut()`  (will need to transpose dimensions to get the color axis from last to first)

Reply:
test image updated to use astronaut image from scikit-image. Additional test cases included.